### PR TITLE
Update quick_install.sh

### PR DESCRIPTION
**Pull request by tanvithakur94**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Type of changes**
Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

**Feature/Issue validation/testing**:

Please describe the tests that you ran to verify your changes and relevant result summary. Provide instructions so it can be reproduced.
Please also list any relevant details for your test configuration.

- [ ] Test A
- [ ] Test B

- Logs

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Checklist**:

- [ ] Have you added unit/e2e tests that prove your fix is effective or that this feature works?
- [ ] Has code been commented, particularly in hard-to-understand areas?
- [ ] Have you made corresponding changes to the documentation?

**Release note**:

```release-note

```

### CODE REVIEW

File: hack/quick_install.sh [hunk 1]
```diff
@@ -122,7 +122,7 @@ kubectl apply -f https://github.com/kserve/kserve/releases/download/${KSERVE_VER
 
 # Install KServe built-in servingruntimes
 kubectl wait --for=condition=ready pod -l control-plane=kserve-controller-manager -n kserve --timeout=300s
-kubectl apply -f https://github.com/kserve/kserve/releases/download/${KSERVE_VERSION}/kserve-runtimes.yaml
+#kubectl apply -f https://github.com/kserve/kserve/releases/download/${KSERVE_VERSION}/kserve-runtimes.yaml
```
Comment (Jooho) [inline hack/quick_install.sh]: It should work so please uncomment this line.

**Review by Jooho — CHANGES_REQUESTED** [review]
I think you already know the reason why this failed.
Looks `kserve-cluster-resouces.yaml` file name was changed to `kserve-runtimes.yaml` in releases. 
so we can fix this issue with changing the filename.

File: hack/quick_install.sh (hunk 1 above)
Comment (tanvithakur94) [reply hack/quick_install.sh]: Thanks. I made the change.

**Comment by yuzisun** [comment]
it is updated in https://github.com/kserve/kserve/commit/6f4bf79e4dd8bd15fb920508b2652bb20222ad9f

**Pull request closed without merging**